test/image/mocks/gl2d_fill_trace_tozero_order.json

```diff
@@ -0,0 +1,105 @@
+{
+  "data": [
+    {
+      "x": [null, 3     ,        3,           null, null
+      ],
+	"y": [ 25,30,35,40        ,50      ],
+	"type": "scattergl",
+	"fill": "tozerox",
+	"mode": "none",
+	"showlegend": false, "xaxis": "x", "yaxis": "y" ,
+	"fillcolor": "#FF69B4",
+	"line": { "color": "#FF69B4" }
+    },
+    {
+      "x": [
+        null,
+        null,
+        null,
+          5,
+	  5
+      ],
+	"y": [ 25,30,35,40        ,50      ],
+	"type": "scattergl",
+	"fill": "tozerox",
+	"mode": "markers+lines",
+	"showlegend": false, "xaxis": "x", "yaxis": "y" ,
+	"fillcolor": "#202ff0",
+	 "line": { "color": "#101ff0" }
+    },
+    {
+      "x": [
+        1,
+        2,
+        3,
+          4,
+	  5
+      ],
+      "y": [ null,       50,        50,null    ,null
+      ],
+	"type": "scattergl",
+	"fill": "tozeroy",
+	"mode": "none",
+	"showlegend": false,  "xaxis": "x2", "yaxis": "y2" ,
+	"fillcolor": "#90ff70",
+	"line": { "color": "#90ff70" }
+    },
+    {
+      "x": [
+        1,
+        2,
+        3,
+          4,
+	  5
+      ],
+	"y": [null
+	    ,null
+	    ,null
+        ,
+        50,
+        50
+      ],
+	"type": "scattergl",
+	"fill": "tozeroy",
+	"mode": "markers+lines",
+	"showlegend": false,   "xaxis": "x2", "yaxis": "y2" ,
+	"fillcolor": "#f02f20",
+	 "line": { "color": "#f01f10" }
+    },
+    {
+	"x": [ 1,  1,  2,  3,  4,  5,5
+      ],
+	"y": [ 0,25,25,25,25,25,0
+      ],
+	"type": "scattergl",
+	"fill": "tozeroy",
+	"mode": "markers+lines",
+	"showlegend": false,    "xaxis": "x3", "yaxis": "y3" ,
+	"fillcolor": "#f02f20",
+	 "line": { "color": "#f01f10" }
+    },
+    {
+	"x": [ 1,  2,  2,  3,  4,  4,5
+      ],
+	"y": [ 0,  0,50,50,50,  0,0
+      ],
+	"type": "scattergl",
+	"fill": "tozeroy",
+	"mode": "none",
+	"showlegend": false,    "xaxis": "x3", "yaxis": "y3" ,
```

Comment:
Can you turn on `showlegend` in these mocks, to help us determine the trace ordering? Thanks!